engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -131,27 +131,26 @@ def _json_chat_output_formatter(token: Token, first_token: bool,
         logprobs = None
         parameters = details.get("parameters", {})
         if parameters.get("logprobs"):
-            logprobs = [
-                {
-                    "token":
-                    t.get("text"),
-                    "logprob":
-                    t.get("log_prob"),
-                    "bytes":
-                    (b := [ord(c)
-                           for c in t.get("text")] if t.get("text") else None),
-                    "top_logprobs":  # Currently only support 1 top_logprobs
-                    [{
-                        "token": t.get("text"),
-                        "logprob": t.get("log_prob"),
-                        "bytes": b
-                    }]
-                } for t in details.get("tokens", [])
-            ]
+            logprobs = {
+                "content": [{
+                        "token":
+                            t.get("text"),
+                        "logprob":
+                            t.get("log_prob"),
+                        "bytes":
+                            (b := [ord(c)
```

Comment:
will `ord` ever throw an exception? what happens if there is no utf-8 representation for the character?

Reply:
This is from the output of the model, and detokenized from token_id. I guess will be very unlikely.